src/embed_html.htl

```diff
@@ -1 +1,3 @@
+<div class="embed embed-internal embed-internal-${content.meta.basename} embed-internal-${content.meta.dirame}-${content.meta.basename}">
```

Comment:
```suggestion
<div class="embed embed-internal embed-internal-${content.meta.basename} embed-internal-${content.meta.dirame}">
```

Nothing I can do with `embed-internal-${content.meta.dirame}-${content.meta.basename}` classname

Reply:
Idea is to be able to address and style all promotions embeds, which is not doable if the class contains each filename.